src/main/resources/l10n/JabRef_es.properties

```diff
@@ -982,9 +980,6 @@ Doing_a_cleanup_for_%0_entries...=Haciendo_la_limpieza_de_%0_entradas
 No_entry_needed_a_clean_up=Ninguna_entrada_necesitó_de_limpieza
 One_entry_needed_a_clean_up=Sólo_una_entrada_necesitó_limpieza
 %0_entries_needed_a_clean_up=%0_entradas_necesitarion_limpieza
-Error_downloading_file_'%0'=Error_descargando_el_archivo_'%0'
```

Comment:
I ran `./gradlew test` locally and it gave me obsolete and missing translation keys which I removed/added to specific files. This translation in particular should not be in use anymore.